alpaca_trade_api/rest.py

```diff
@@ -269,10 +269,10 @@ def list_orders(self,
 
     def submit_order(self,
                      symbol: str,
-                     qty: int,
-                     side: str,
-                     type: str,
-                     time_in_force: str,
+                     qty: float = None,
```

Comment:
If a user had a variable typed as `int` for input I think there may be some type checkers that may throw an error/warning since this parameter now has a different type than before. 
Not sure how big of an issue this is tbh (or whether this really is the case for some type checkers e.g. mypy).

Reply:
Not sure about other type checkers, but mypy recognizes [duck type compatibility](https://mypy.readthedocs.io/en/latest/duck_type_compatibility.html), so it treats `int` objects as valid `float` objects